.github/issue_template.md

```diff
@@ -1,3 +1,10 @@
+<!-- Title should describe the feature or problem from the user's perspective, and/or be directly actionable for someone taking on the issue -->
+
+<!-- FILL OUT THE CHECKLIST BELOW -->
```

Comment:
Add several sections to guide input....
1) Detailed description
2) Reproduction Steps
3) What was expected to happen
4) Add screenshots or other useful content

Reply:
Updated 👍